aws/rust-runtime/aws-types/src/environment.rs

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+//! Abstractions for testing code that interacts with the process environment:
+//! - Determining the current time
+//! - Reading environment variables
+//! - Reading from the file system
```

Comment:
How does this look in the module summary line in the generated docs when it has bullet points like this?

Reply:
just renders as bullets on the module page